Add new explore/observability related features to whats new docs (#27540)
grafana/grafana: docs/sources/guides/whats-new-in-v7-2.md

- [**Table column filters**]({{< relref "#table-column-filters" >}})
- [**New and improved transforms**]({{< relref "#new-and-improved-transforms" >}})
- [**Field override matchers**]({{< relref "#field-override-matchers" >}})

> - [**Query inspector in Explore**]({{< relref "#query-inspector-in-explore" >}})

Reviewer: Just a reminder, every feature should have a link to updated documentation.

Author: Added

You can now add override rules that use a regex matcher and overrides that match on field type.

> ## Query inspector in Explore

Reviewer: ```suggestion
## Inspect queries in Explore
```

> You can enjoy all the details query inspector gave you in dashboards now in explore as well. You can open query inspector tab with the button next to query history.

Reviewer: Capitalize the name of the tab. Match the UI.

Author: Done

## $__rate_interval for Prometheus

> You can now use the new variable `$__rate_interval` in Prometheus for rate functions mainly. `$__rate_interval` in general is one scrape interval larger than `$__interval` but is never smaller than four times the scrape interval (which is 15s by default). See the [Prometheus datasource]({{< relref "../features/datasources/prometheus.md#using-__rate_interval-variable" >}}) for more details.

Reviewer: ```suggestion
You can now use the new variable `$__rate_interval` in Prometheus for rate functions mainly. `$__rate_interval` in general is one scrape interval larger than `$__interval` but is never smaller than four times the scrape interval (which is 15s by default). See the [Prometheus data source]({{< relref "../features/datasources/prometheus.md#using-__rate_interval-variable" >}}) for more details.
```

Reviewer: Per the Grafana style guide, "data source" is always two words.

## Loki toggling parsed fields

> With this awesome contribution from one of our community members you can now toggle parsed fields in Explore.

Reviewer: ```suggestion
With this awesome contribution from one of our community members, you can now toggle parsed fields in Explore.
```

> ## Loki toggling parsed fields

Reviewer: ```suggestion
## Loki now toggles parsed fields
```

Reviewer 2: It's not Loki toggling the fields. The user can choose which parsed fields should be shown. When a user toggles a parsed field so it's shown, the other fields are being hidden.

Author: @oddlittlebird can you help me find a better title for this feature?

Reviewer: Toggle parsed Loki fields?

Author: Sounds good to me. Let me change that.
